src/tools/crossgen2/ILCompiler.ReadyToRun/Compiler/ReadyToRunCodegenCompilation.cs

```diff
@@ -14,6 +14,7 @@
 using ILCompiler.DependencyAnalysis;
 using ILCompiler.DependencyAnalysis.ReadyToRun;
 using ILCompiler.DependencyAnalysisFramework;
+using System.Collections;
```

Comment:
Nit: Move to a few lines above with the rest of the "using System"'s

Reply:
Actually was an unused import. Removed.